parl/framework/model_base.py

```diff
@@ -18,13 +18,13 @@
 import hashlib
 import paddle.fluid as fluid
 from abc import ABCMeta
-from parl.framework.interface import NetworkInterface
+from parl.layers.layer_wrappers import LayerFunc
 from parl.plutils import *
```

Comment:
Can we remove this line ? 
It seems that we don't have any function imported from parl.utils currently.

Reply:
Here is plutils, we have imported functions from plutils.